neural_structured_learning/examples/notebooks/Adversarial_regularization_for_image_classification save and load the adv model.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<p>Thanks for adding an example notebook. I think it is great to demonstrate the saving and loading functionalities. Please update the title and the overview section to focus on that. Also maybe please rename this file to match the new title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/neural-structured-learning/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>